### PR TITLE
#20 handle %n and %%

### PR DESCRIPTION
**Pull request by MartyHub**

### CODE REVIEW

**Comment by dmarkov** [comment]
@martyhub Thanks, I will find someone to review this PR soon

**Comment by dmarkov** [comment]
@dmzaytsev please review

File: src/test/java/com/jcabi/log/NoArgTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+/**
+ * Test case for no argument specifier in {@link Logger}.
+ * @author Julien Gotteland (julien.gotteland@laposte.net)
+ * @version $Id$
+ */
+public final class NoArgTest {
+
+    /**
+     * New line specifier can be used.
```
Comment (dmzaytsev) [inline src/test/java/com/jcabi/log/NoArgTest.java]: @MartyHub Preformater can handle new line specifier.

File: src/test/java/com/jcabi/log/NoArgTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+     */
+    @Test
+    public void handleNewLine() {
```
Comment (dmzaytsev) [inline src/test/java/com/jcabi/log/NoArgTest.java]: @MartyHub `handlesNewLine()`

File: src/test/java/com/jcabi/log/NoArgTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        final String fmt = "%s%n%s";
+        final Object[] args = {"new", "line"};
+        MatcherAssert.assertThat(
+                Logger.format(fmt, args),
+                Matchers.is(String.format(fmt, args))
+        );
+    }
+
+    /**
+     * Percent specifier can be used.
```
Comment (dmzaytsev) [inline src/test/java/com/jcabi/log/NoArgTest.java]: @MartyHub Preformater can handle percent specifier.

File: src/test/java/com/jcabi/log/NoArgTest.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+     */
+    @Test
+    public void handlePercent() {
```
Comment (dmzaytsev) [inline src/test/java/com/jcabi/log/NoArgTest.java]: @MartyHub `handlesPercent()`

**Comment by dmzaytsev** [comment]
@MartyHub many thanks for your contribution.
please see several comments above.
We are follow [the rule](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names
) for test method names

Also please put test methods into this class https://github.com/jcabi/jcabi-log/blob/master/src/test/java/com/jcabi/log/PreFormatterTest.java

**Comment by dmzaytsev** [comment]
@MartyHub thank you

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@yegor256 ping

**Comment by dmzaytsev** [comment]
@yegor256 please pay attention to this PR
we are waiting for your confirmation for 8 days

**Comment by yegor256** [comment]
@rultor try to merge it

**Comment by dmarkov** [comment]
@dmzaytsev **20 mins** sent to your balance (ID `64032354`), many thanks! It took 233 hours and 37 mins.... you're getting extra minutes here (c=5)... added +20 to your rating, now it is equal to [+1816](http://www.netbout.com/b/35166?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**